### PR TITLE
Document how to rerun watchdog:show every 2 seconds.

### PR DESCRIPTION
**Pull request by weitzman**
Fixes #3523 (in a more generic way).

### CODE REVIEW

**Comment by cspitzlay** [comment]
The merged example does not work. It is missing a "do"

**Comment by weitzman** [comment]
Fixed. Thanks.

File: src/Drupal/Commands/core/WatchdogCommands.php [hunk 1]
```diff
@@ -34,6 +34,8 @@ class WatchdogCommands extends DrushCommands
      *   Show a listing of most recent 10 messages with a severity of notice.
      * @usage drush watchdog:show --type=php
      *   Show a listing of most recent 10 messages of type php
+     * @usage  while sleep 2; drush watchdog:show; done
+     *   Every 2 seconds, show the most recent 10 messages.
```
Comment (amaisano) [inline src/Drupal/Commands/core/WatchdogCommands.php:38]: Doesn't validate or work (mac osx):

image

Comment (weitzman) [reply src/Drupal/Commands/core/WatchdogCommands.php:38]: That was pointed out above and then fixed https://github.com/drush-ops/drush/commit/a9840accc5554d2e628a323ad179e94d2555694e

Comment (amaisano) [reply src/Drupal/Commands/core/WatchdogCommands.php:38]: Ah sorry! Google pointing me to old posts.

**Comment by dahousecat** [comment]
`watch -n2 drush ws` works better for me as doesn't show duplicated rows.

**Pull request merged**